schema/test/unit/computed_columns/project_next_milestone_due_date_test.sql

```diff
@@ -53,19 +62,20 @@ values
 ('PM', false, true);
 
 /** Create mock reporting requirement **/
-insert into cif.reporting_requirement(id, project_id, report_type, report_due_date, submitted_date, comments, reporting_requirement_index, description)
+insert into cif.reporting_requirement(id, project_id, report_type, report_due_date, submitted_date, comments, reporting_requirement_index, description, archived_at)
 overriding system value
 values
-(1, 1, 'GM', '2023-05-20 13:42:59.085 -0800', null, 'milestone 1', 1, 'upcoming milestone'),
-(2, 1, 'AM', '2023-01-23 13:42:59.085 -0800', null, 'milestone 2', 1, 'late milestone'),
-(3, 1, 'PM', '2023-01-16 13:42:59.085 -0800', '2023-01-06 13:42:59.085 -0800', 'milestone 3', 1, 'completed milestone'),
-(4, 2, 'A', '2023-04-15 13:42:59.085 -0800', '2023-02-15 13:42:59.085 -0800', 'milestone 4', 1, 'annual') ,
-(5, 2, 'AM', '2023-08-05 13:42:59.085 -0800', '2023-06-11 13:42:59.085 -0800', 'milestone 5', 1, 'completed milestone'),
-(6, 2, 'GM', '2023-03-02 13:42:59.085 -0800', '2023-02-21 13:42:59.085 -0800', 'milestone 6', 1, 'completed milestone'),
-(7, 2, 'GM', '2023-09-01 13:42:59.085 -0800', '2023-04-018 13:42:59.085 -0800', 'milestone 7', 1, 'completed milestone'),
-(8, 3, 'AM', '2023-04-22 13:42:59.085 -0800', null, 'milestone 8', 1, 'upcoming milestone'),
-(9, 3, 'Q', '2023-06-15 13:42:59.085 -0800', null, 'milestone 9', 1, 'upcoming milestone'),
-(10, 3, 'GM', '2023-04-04 13:42:59.085 -0800', null, 'milestone 10', 1, 'upcoming milestone');
+(1, 1, 'GM', '2023-05-20 13:42:59.085 -0800', null, 'milestone 1', 1, 'upcoming milestone', null),
+(2, 1, 'AM', '2023-01-23 13:42:59.085 -0800', null, 'milestone 2', 1, 'late milestone', null),
+(3, 1, 'PM', '2023-01-16 13:42:59.085 -0800', '2023-01-06 13:42:59.085 -0800', 'milestone 3', 1, 'completed milestone', null),
+(4, 2, 'A', '2023-04-15 13:42:59.085 -0800', '2023-02-15 13:42:59.085 -0800', 'milestone 4', 1, 'annual', null) ,
```

Comment:
It looks like project 1 tests that the coalesce returns `next_due_date`, and 2 is testing the coalesce for `last_submitted_date`. Could we add that to the test descriptions for clarity?